djed-sdk/dist/umd/index.js

```diff
@@ -1,6 +1,6 @@
 (function (global, factory) {
-  typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('web3')) :
-  typeof define === 'function' && define.amd ? define(['exports', 'web3'], factory) :
+  typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('web3'), require('ethers')) :
```

Comment:
Why are we needing "ethers" now?  It should be possible for us to depend on only one of "web3" and "ethers", shouldn't it?

Reply:
Then maybe we should consider using only ethers and not web3?

Reply:
@ceilican I've been reviewing discussions on the ethers GitHub page and discovered that instead of incorporating the whole ethers library, it's possible to selectively use just the BigNumber module. This approach allows us to maintain our current usage of the web3 library across many pages without disruption. However, for the SDK specifically, adopting the BigNumber module from ethers seems advantageous.

Interestingly, our frontend already utilizes ethers for various components alongside the web3 library. Given that our project is bootstrapped with Create React App (CRA), it inherently benefits from tree shaking capabilities. This means that any unused code, including parts of ethers or web3 that aren't actively utilized, will be automatically excluded from the final bundle. This optimization ensures efficiency despite the coexistence of both libraries in our project.